### PR TITLE
Added project named Headline-Hub under Node-JS-Projects

### PR DESCRIPTION
**Pull request by 1simransaini**
Closes: #267

- **Title** : Headline-Hub
- **Name:** :  Simran Saini
- **Idenitfy yourself:** : GSSOC Contributor

### Describe the add-ons or changes you've made 📃
Added project named Headline-Hub, it is a dynamic news platform built using Node.js, Express, and EJS. It provides users with the latest news articles from various sources, presenting them in an organized and user-friendly interface. Latest articles are fetched from the NewsAPI website.

### How Has This Been Tested? ⚙️

I tested it at my local system.

### Screenshots 📷

Screenshot (14293)

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Kindly go through it.

### CODE REVIEW

File: Node-JS-Projects/Basic/Headline-Hub/screenshot.webp.webp
Comment (Avdhesh-Varshney) [inline Node-JS-Projects/Basic/Headline-Hub/screenshot.webp.webp:1]: name is not `screenshot.webp` rename it with `screenshot`. Here `webp` is the name of file extension only.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by GAVINESHWAR** [comment]
@1simransaini  I am getting error while using nom start

PS C:\Users\Avineshwar G\Documents\internship\opensource\WebMasterLog> npm start
npm ERR! Missing script: "start"
npm ERR!
npm ERR! Did you mean one of these?
npm ERR! npm star # Mark your favorite packages
npm ERR! npm stars # View packages marked as favorites
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR! npm run

npm ERR! A complete log of this run can be found in: C:\Users\Avineshwar G\AppData\Local\npm-cache_logs\2024-06-21T05_25_24_324Z-debug-0.log

Can you say how you open the project

File: Node-JS-Projects/Basic/Headline-Hub/screenshot.webp.webp
Comment (1simransaini) [reply Node-JS-Projects/Basic/Headline-Hub/screenshot.webp.webp:1]: @Avdhesh-Varshney I have renamed `screenshot.webp.webp` to `screenshot.webp`. 
Kindly review it.

**Comment by 1simransaini** [comment]
@GAVINESHWAR Maybe you have not install dependencies, also you need to first locate to the correct directory, In order to run the project successfully, please follow these steps:

- **Navigate to the Project Directory:**
  Make sure you are in the correct directory where the package.json file is located. Path should be:
  ```
  cd WebMasterLog\Node-JS-Projects\Basic\Headline-Hub
  ```
  (maybe in your case it is `cd C:\Users\Avineshwar G\Documents\internship\opensource\WebMasterLog\Node-JS-Projects\Basic\Headline-Hub`)

- **Install Dependencies:**
  Ensure all the dependencies listed in package.json are installed:
  ```
  npm install
  ```
- **Now run the Start Script:**
  ```
  npm start
  ```
  If npm start doesn't work, you can manually run the start command as:
  ```
  nodemon app.js
  ```

**Review by Avdhesh-Varshney — APPROVED** [review]
@1simransaini PR Approved 🎉

**Pull request merged**